app/admin.js

```diff
@@ -220,6 +271,12 @@ const handleGenerateKeys = function(args) {
   fs.writeFileSync(args.output, JSON.stringify(keys, null, 2));
 };
 
+const handleGenerateHDSeed = function() {
+  const XLM_SEED_LENGTH = 64;
+
+  console.log(crypto.randomBytes(XLM_SEED_LENGTH).toString('hex'));
```

Comment:
Do we want to be providing randomness generation? I thought we were mostly going to leave any of generation stuff to them outside this repo

Reply:
Or put it in BitGo CLI? I also might be entirely misinterpreting what this is used for

Reply:
BitGo CLI has a method for generating a random BIP32 keypair, but no such utility for Stellar. Our Readme does specify how to generate a master BIP32 keypair with BitGo CLI, so I added this for convenience. This command is plenty secure for random seed generation given enough entropy (we'll probably provide an Infinite Noise to KRS providers)